Content.Server/Gravity/EntitySystems/GravitySystem.cs

```diff
@@ -137,15 +137,8 @@ private void ScheduleGridToShake(GridId gridId, uint shakeTimes, GravityComponen
             {
                 _gridsToShake[gridId] = shakeTimes;
             }
-            // Play the gravity sound
-            foreach (var player in _playerManager.GetAllPlayers())
-            {
-                if (player.AttachedEntity == null
-                    || player.AttachedEntity.Transform.GridID != gridId)
-                    continue;
 
-                SoundSystem.Play(Filter.Pvs(player.AttachedEntity), comp.GravityShakeSound.GetSound(), player.AttachedEntity);
-            }
+            SoundSystem.Play(Filter.Broadcast(), comp.GravityShakeSound.GetSound(), AudioParams.Default.WithVolume(-2f));
```

Comment:
Hmm, this means people not on the station will be able to hear the gravity gen sound too

Reply:
You can't hear sound in space